app/src/main/java/com/zeapo/pwdstore/PasswordEntry.kt

```diff
@@ -81,6 +87,36 @@ class PasswordEntry(private val content: String) {
         return null
     }
 
+    private fun findOtpDigits(decryptedContent: String): String? {
+        decryptedContent.split("\n".toRegex()).forEach { line ->
+            if (line.startsWith("otpauth://totp/")) {
+                return Uri.parse(line).getQueryParameter("digits")
+            }
+            if (line.startsWith("otpauth://hotp/")) {
+                return Uri.parse(line).getQueryParameter("digits")
```

Comment:
Thanks. I'm not quite familiar with Kotlin. After trying out `when`, I don't think it's suitable in this place, as only if it failed to return a value after the `forEach`, it can then return a default value like "6". So I then decide to merge these two if branch into one.

Reply:
Actually, `otpauth://` URI is usually added using [pass-otp](https://github.com/tadfisher/pass-otp), and a line starts with `otpauth://` is their default scheme. Previously implemented functions like `findTotpSecret` are actually done in this way. What I've done is just mock their implementing method.